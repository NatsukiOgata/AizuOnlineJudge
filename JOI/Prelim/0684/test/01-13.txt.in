5 100
63 87 16 20 41
35 7 80 43 72 28 37 27 94 92 3 41 20 54 42 18 89 100 39 37 86 52 43 26 62 87 78 5 72 19 79 95 24 35 46 71 23 93 59 46 70 19 21 70 93 25 82 73 4 72 77 57 91 99 92 36 99 12 34 80 97 37 82 94 74 99 93 71 91 36 21 47 84 75 58 48 63 2 91 60 44 89 61 2 63 93 79 87 71 49 27 45 19 28 82 68 94 71 53 53
