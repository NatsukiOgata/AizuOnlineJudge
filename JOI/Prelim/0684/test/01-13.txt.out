20
41
63
87
